src/store/modules/site_config.js

```diff
@@ -187,6 +187,13 @@ const getters = {
     return fwo
   },
 
+  // Available automatic 3-color image options
+  auto_color_options: (state, getters) => {
+    const fwo = getters.selected_filter_wheel_config.settings?.auto_color_options
+    if (fwo == undefined) return [[]]
```

Comment:
Generally a good idea to use the strict equality operator `===` to check if something is undefined. The loose equality check `==` will equate `undefined` and `null`, which usually isn't intended behavior. (I know, my mistake in the method above too.)

Also, while the filter wheel options expects a different format (the first array element is an array of keys, so an empty value still needs one element). But the auto_color_options is just a simple array of values. So the empty value should just be a simple `[]` rather than `[[]]`.